### DISCUSSION TITLE
Did text selection just get improved?

### DISCUSSION THREAD

**Discussion by kefabean** [zed-industries/zed#6654] [Feature Requests]
I thought it was me, but I always found text selection to be a little inaccurate under Zed, particularly at smaller font sizes.

I noticed on the latest stable version (v0.111.6) this seems to be resolved. I'm not entirely sure, but if someone can confirm this is excellent news. THANK-YOU team!

(the only related issue I could find was this one that seems way too old, and inconclusive https://github.com/zed-industries/community/issues/528)

**Comment by widersky** [comment]
Didn't have such issue but it's great you have it fixed :)

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
